pages/common/vcsh.md

```diff
@@ -0,0 +1,27 @@
+# vcsh
+
+> Version Control System for $HOME - multiple Git repositories in $HOME.
+
+- Execute git command on managed repository:
+
+`vcsh {{repository_name}} {{git_command}}`
+
+- Initialize (empty) repository:
+
+`vcsh init {{repository_name}}`
+
+- Clone repository into custom name:
+
+`vcsh clone {{git_url}} {{repository_name}}`
+
+- List all managed repositories:
+
+`vcsh list`
+
+- Pull/pull all managed repositories from/to remotes:
```

Comment:
I'm assuming this should be `Push/pull all...`?

Reply:
Thank you for noticing! I fixed it;